automation/tests/test_slack.py

```diff
@@ -0,0 +1,18 @@
+from .. import config
+from ..utils import slack
+
+import pytest
+
+TEST_USER_EMAIL = config.Config.load().slack.test_user_email
+TEST_USER_ID = config.Config.load().slack.test_user_id
+
+
+@pytest.mark.skip(reason="Hits the slack API")
+def test_users_lookupByEmail():
+    client = slack.SlackClient()
+
+    user = client.users_lookupByEmail(email=TEST_USER_EMAIL)
+    assert user.id == TEST_USER_ID
+
+    user = client.users_lookupByEmail(email="<invalid email>")
+    assert user is None
```

Comment:
```suggestion
@pytest.fixture
def test_config():
    return config.Config.load()


@pytest.fixture
def test_user_email(test_config):
    return test_config.slack.test_user_email


@pytest.fixture
def test_user_id(test_config):
    return test_config.slack.test_user_id


@pytest.mark.skip(reason="Hits the slack API")
def test_users_lookupByEmail(test_user_email, test_user_id):
    client = slack.SlackClient()

    user = client.users_lookupByEmail(email=test_user_email)
    assert user.id == test_user_id

    user = client.users_lookupByEmail(email="<invalid email>")
    assert user is None
```

Reply:
(This makes this module safe to load when the test is skipped, otherwise it raises on module import if you have the wrong config)

Reply:
Integrated